server/controllers/article.js

```diff
@@ -81,14 +104,10 @@ export const getArticle = async (req, res) => {
 };
 
 export const updateArticle = async (req, res) => {
-  const { userId } = req;
-  const { id } = req.params;
-
   const { title, body } = req.body;
+
   try {
-    const article = await Article.findOne({
-      where: { id, userId }
-    });
+    const article = await findArticle(req);
```

Comment:
I believe you can calculate the reading time directly because the title and body is already available to you, there is no need to go to the DB